combined_iterator_integration_test.go

```diff
@@ -39,6 +40,7 @@ func testCombinedIterator(ctx context.Context, t *testing.T, is *is.I) (common.I
 		serverID:            testutils.ServerID,
 		mysqlConfig:         config,
 		disableCanalLogging: true,
+		mysqlVer:            gover.Must(gover.NewVersion("8.0")),
```

Comment:
can we have this as a constant or a var somewhere? it's used a lot in the code and the tests, so could be useful. [doesn't have to be only one constant for all the files, tests can have their own]